tests/trainer/test_dataloaders.py

```diff
@@ -1066,6 +1066,41 @@ def test_dataloaders_load_every_epoch(tmpdir):
         assert call['name'] == expected
 
 
+@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
+def test_dataloaders_load_every_n_epochs(tmpdir):
+
+    model = EvalModelTemplate()
```

Comment:
Better if you use `BoringModel` (its faster)

```suggestion
    model = BoringModel()
```